### PR TITLE
Restructure bundling, added styles

### PR DESCRIPTION
**Pull request by b30wulffz**

### CODE REVIEW

File: src/custom_components/slider.scss [hunk 1]
```diff
@@ -1,8 +1,7 @@
-.virtual-slider {
+.v-slider {
```
Comment (raj-vlabs) [inline src/custom_components/slider.scss:1]: Why do we have a slider2 which we are including and a slider which we are not including?
Unless there's a good reason, please remove slider2 and add slider to the _all file

File: src/variable_tweaks.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// @import url("https://fonts.googleapis.com/css?family=Nunito:400,700");
```
Comment (raj-vlabs) [inline src/variable_tweaks.scss]: Please use a more formal name for this file. variable_definition or variable_redefine, or even variables is a good enough name.

File: src/custom_components/_all.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@import "./slider2.scss";
```
Comment (raj-vlabs) [inline src/custom_components/_all.scss]: Please include files in alphabetical order unless if there's some dependency between them

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: src/custom_components/slider.scss (hunk 1 above)
Comment (b30wulffz) [reply src/custom_components/slider.scss:1]: Resolved

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**